charter.md

```diff
@@ -0,0 +1,105 @@
+- Continually improving release and development processes
+  - Work closely with SIG Contributor Experience to define and build tools to facilitate release process (e.g. dashboards)
+  - Work with downstream communities responsible for packaging Kubernetes releases
+  - Work with other SIGs to agree upon the responsibilities of their SIG with respect to the release
+  - Define and collect metrics related to the release in order to measure progress over each release
+  - Facilitate release retrospectives in conjunction with SIG PM
+- Collaboration with downstream communities which build artifacts from Kubernetes releases
+  - Ensure the appropriate level of integration with publishing the OSS build artifacts
+
+### Out of scope
+
+#### Support
+
+SIG Release itself is not responsible for end user support or creation of patches for support streams. There are support forums for end users to ask questions and report bugs, subject matter experts in other SIGs triage and address issues and when necessary mark bug fixes for inclusion in a patch release.
+
+## Roles and Organization Management
+
+This SIG adheres to the Roles and Organization Management outlined in [sig-governance] and opts-in to updates and modifications to [sig-governance].
+
+Specifically, the common guidelines (see: [sig-governance]) for continuity of membership within roles in the SIG are followed.
+
+### Deviations from [sig-governance]
+
+- SIG Release does not have top-level SIG Technical Leads; SIG Release Chairs can serve in both an operational and technical capacity
+- SIG Release subprojects have subproject chairs
+
+#### SIG Membership
+
+Additionally, SIG Release has a concept of membership. SIG members can be occasionally called on to assist with decision making, especially as it relates to gathering historical context around existing policies.
+
+While a strict definition of membership has yet to be established, we can initially seed a set of members from the list of Release Team members from Kubernetes 1.10 through to Kubernetes 1.13.
+
+SIG Release Members should be the first group of community members considered when seeking to fill additional roles across SIG Release proper and SIG Release subprojects, except where explicitly defined in other policy.
+
+### Subproject Creation
+
+- Subprojects must be created by [KEP] proposal and accepted by [lazy-consensus] with fallback on majority vote of
+  SIG Chairs.  The result *SHOULD* be supported by the majority of SIG members.
+  - KEP *MUST* establish subproject chairs
+  - [sigs.yaml] *MUST* be updated to include subproject information and OWNERS files with subproject chairs
+  - Where subprojects processes differ from the SIG governance, they must document how
+    - e.g. if subprojects release separately - they must document how release and planning is performed
+
+As of the initial draft of this charter, the following are the proposed subprojects / WGs for SIG Release:
+
+#### Release Team (Subproject)
```

Comment:
Subprojects are not typically called out in the charter, b/c they are subject to change.  The charter spells out the mission and details of the subproject(s) are simply added to the community repo and align with that mission.